packages/synthetic-chain/public/upgrade-test-scripts/run_eval.sh

```diff
@@ -17,11 +17,12 @@ echo "[$PROPOSAL] Agd started. Running CoreEval submission."
 cd /usr/src/proposals/"$PROPOSAL/" || fail "Proposal $PROPOSAL does not exist"
 
 if [ -f "eval.sh" ]; then
-    # this is what the script used to do. Also allows a proposal to override how they are eval-ed
+    # Allows a proposal to control its evaluation; used in particular for param-change proposals.
```

Comment:
Maybe keep part of the comment that core eval with custom logic can use this too?

Reply:
Done.